cyberark-secrets-provider-for-kubernetes/templates/secrets-provider.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: {{ include "conjur.deploy-api" . }}
+kind: Job
+metadata:
+  name: {{ .Release.Name }}
+  namespace: {{ .Release.namespace }}
+spec:
+  template:
+    spec:
+      serviceAccountName: {{ .Values.rbac.serviceAccount.name }}
+      containers:
+      - image: {{ .Values.secretsProvider.image }}:{{ .Values.secretsProvider.tag }}
+        imagePullPolicy: IfNotPresent
```

Comment:
I would make all `imagePullPolicy` configurable. It's a highly useful parameter in my opinion

Reply:
I'd strongly +1 this

Reply:
Okay, I will add 
`imagePullPolicy: IfNotPresent` in the values.yaml and the user will be able to override this one via `custom-values.yaml`